Evaluation/Input/evaluation_plan.json

```diff
@@ -1,9 +1,9 @@
 {
-    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json",
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.0/OSP_Qualification_Plan_Schema.json",
     "Projects": [
         {
             "Id": "Compound",
-            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Evaluation-plan-template/master/Compound.json",
+            "Path": "../../Compound.json",
```

Comment:
Compound.json comes from the same repository and should be referenced with relative path, not from Github!